### ISSUE TITLE
Add Gifs to documentation

### ISSUE THREAD

**Issue by wadlo** [wadlo/open-top-down-engine#3]
There's some cool demos in this project, but there aren't any gifs. People don't really know what this code does.

**Comment by mvoorhes** [comment]
Working on this right now but am having trouble getting anything to load through the demos. I should mention I'm new to godot so I'm pretty unfamiliar with this stuff. Also am developing on mac if that makes a difference.

Right now I'm getting this warning 

`modules/gltf/register_types.cpp:73 - Blend file import is enabled in the project settings, but no Blender path is configured in the editor settings. Blend files will not be imported.`

This is what is currently showing up in godot

<img width="1169" alt="Screenshot 2023-08-25 at 12 40 11 PM" src="https://github.com/wadlo/open-top-down-engine/assets/100326501/e3eb6fdd-e7d4-46c7-be47-321bfb30d454">


Are there files missing from the source code that I should know about in order to run the demos?

**Comment by wadlo** [comment]
Hey! I found windows has a bug with git that was causing issues. I had to rename the folder "usable" to "Usable". If you're on Windows, it's probably best you just delete the repo and reclone it.

Anyway, you probably have to run "build" before actually running the game, then the buttons should start creating units on the left and right side of the screen. This might be easier to discuss over discord. Here's my server link: https://discord.gg/y2MkrQyx
